offchain/authority-claimer/src/config/json.rs

```diff
@@ -0,0 +1,29 @@
+// (c) Cartesi and individual authors (see AUTHORS)
+// SPDX-License-Identifier: Apache-2.0 (see LICENSE)
+
+use rollups_events::{Address, Hash};
+use serde::{de::DeserializeOwned, Deserialize};
+use snafu::ResultExt;
+use std::{fs::File, io::BufReader, path::PathBuf};
+
+use crate::config::error::{
+    AuthorityClaimerConfigError, JsonParseSnafu, ReadFileSnafu,
+};
+
+#[derive(Clone, Debug, Deserialize)]
+pub(crate) struct DappDeployment {
+    #[serde(rename = "address")]
+    pub dapp_address: Address,
+
+    #[serde(rename = "blockHash")]
+    pub dapp_deploy_block_hash: Hash,
+}
```

Comment:
I'm not too fond of this being copied here. Each service has its way of setting up the DApp deployment data. Can you create an issue to unify the DApp configuration across all services?

Reply:
We could also unify this config with DAppMetadata from rollups-events.